tensorboard/plugins/scalar/summary_test.py

```diff
@@ -100,42 +57,70 @@ def test_explicit_display_name_and_description(self):
     metadata.parse_plugin_metadata(content)
 
   def test_float_value(self):
-    pb = self.compute_and_check_summary_pb('a', 1.13)
+    pb = self.scalar('a', 1.13)
     value = tf.make_ndarray(pb.value[0].tensor).item()
     self.assertEqual(float, type(value))
     self.assertNear(1.13, value, 1e-6)
 
   def test_int_value(self):
     # ints should be valid, but converted to floats.
-    pb = self.compute_and_check_summary_pb('a', 113)
+    pb = self.scalar('a', 113)
     value = tf.make_ndarray(pb.value[0].tensor).item()
     self.assertEqual(float, type(value))
     self.assertNear(113.0, value, 1e-6)
 
   def test_bool_value(self):
     # bools should be valid, but converted to floats.
-    pb = self.compute_and_check_summary_pb('a', True)
+    pb = self.scalar('a', True)
     value = tf.make_ndarray(pb.value[0].tensor).item()
     self.assertEqual(float, type(value))
     self.assertEqual(1.0, value)
 
-  def test_string_value_in_op(self):
-    with six.assertRaisesRegex(self, Exception, r'Cast str.*float'):
-      with tf.Session() as sess:
-        sess.run(summary.op('a', tf.constant("113")))
+  def test_string_value(self):
+    with six.assertRaisesRegex(self, Exception, r'Cast string.*float'):
```

Comment:
test is failing in Python3 because dtype of string is.... str96?! 😨

Reply:
Done. My bad for thinking I could relax that regex; comment added.